README.md

```diff
@@ -97,6 +100,18 @@ export VK_COMPILE_TIME_LOG=<log-file-path>
 
 To check if the layers are enabled, you can run a sample Vulkan application (such as `vkcube`) and look for the generated layer log files, or check the [Vulkan loader logs](https://github.com/KhronosGroup/Vulkan-Loader/blob/master/docs/LoaderInterfaceArchitecture.md#table-of-debug-environment-variables).
 
+### Log formats:
+
+To visualize the logs, we use multiple trace viewers such as the analysis scripts in the [scripts directory](./scripts/) and Perfetto (https://ui.perfetto.dev/). To suit the input format requirements of each logger, we support logging in various formats such as CSV and Chrome Trace Event. To learn more about the Trace Event format, see:
```

Comment:
I'm not a fan of this description:
1. Doesn't get to the point quickly enough.
2. Format requirements are irrelevant to end users and how we suit them doesn't matter.
3. Common logs are not CSV files.

Instead, should we list all supported formats, create a paragraph describing each format, what it is useful for, what are the properties, which flag enables it, etc.?